gui/default/index.html

```diff
@@ -81,6 +81,8 @@
             <li class="divider" aria-hidden="true"></li>
             <li><a href="" ng-click="advanced()"><span class="fas fa-fw fa-cogs"></span>&nbsp;<span translate>Advanced</span></a></li>
             <li><a href="" ng-click="logging.show()"><span class="far fa-fw fa-file-alt"></span>&nbsp;<span translate>Logs</span></a></li>
+            <li class="divider" aria-hidden="true"></li>
```

Comment:
One more thing. These items should only be shown when debugging is enabled, otherwise the link will anyway not work. Also not sure why the font size adjustment here?

Reply:
Font size is just for icon, may be adjustment in JS is appropriate, I will check it.